src/main/java/io/lenses/connect/smt/header/InsertWallclockDateTimePart.java

```diff
@@ -59,11 +62,19 @@ void setInstantF(Supplier<Instant> instantF) {
                   + Arrays.stream(DateTimePart.values())
                       .map(Enum::name)
                       .reduce((a, b) -> a + ", " + b)
-                      .orElse(""));
+                      .orElse(""))
+          .define(
+              ConfigName.TIMEZONE,
+              ConfigDef.Type.STRING,
+              "UTC",
+              ConfigDef.Importance.HIGH,
+              "The timezone to use.");
 
   interface ConfigName {
     String HEADER_NAME = "header.name";
     String DATE_TIME_PART = "date.time.part";
+
+    String TIMEZONE = "timezone";
```

Comment:
shouldn't all of those config names be `static final`?

Reply:
probably.